compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -409,6 +409,8 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager, bool packPeel) {
   else
     passManager.addPass(createAMDAIEPackToDmaPass());
 
+  passManager.addPass(createCanonicalizerPass());
```

Comment:
I would only add these to the else part of the above if/else to not unintentionally mess with the peel-pack flow.